src/lobbyconnection.py

```diff
@@ -2275,6 +2275,9 @@ def command_game_host(self, message):
 
 
     def command_modvault(self, message):
+        if "type" not in message:
+            # TODO: message, exception if invalid
+            return
```

Comment:
Related to FAForever/client#187 in that if we don't reply here, the client may end up killing the users disk space.

This command isn't executed through the offending `SecondaryServer` class, but we should still ensure a request always gets a reply.